pyproject.toml

```diff
@@ -69,6 +70,7 @@ django_ninja = ["django", "django-ninja", "requests"]
 django_rest_framework = ["django", "djangorestframework", "requests"]
 fastapi = ["fastapi", "starlette", "httpx"]
 flask = ["flask", "requests"]
+litestar = ["litestar", "httpx"]
```

Comment:
Do you need httpx or are you just being explicit or wanting diff. version? Just wondering

https://github.com/litestar-org/litestar/blob/fe7214378553a88701222e4895746fe15f6844ab/pyproject.toml#L35C3-L35C17

Reply:
httpx is used by the async ApitallyClient, so just being explicit here.